<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_registrate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Sitka&quot;;
font: 700 24pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Registrate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="captura_username">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre de usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="captura_pass">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="captura_pass_confirm">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirmar contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crear_cuenta">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Cuenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="alerta_error">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(255, 0, 0)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="captura_mail">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Correo electrónico</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
